components/google_sheets/sources/new-comment/new-comment.mjs

```diff
@@ -1,16 +1,30 @@
-import httpBase from "../common/http-based/sheet.mjs";
+import app from "../../google_sheets.app.mjs";
```

Comment:
My recommendation here is to leave all the webhook sources as they are and if you want to give support to polling, please create a new one. for example there is one named `new-row-added` which it is webhook based and its equivalent is `new-row-added-polling` so if you find any of the polling sources are failing you can fix them instead